### ISSUE TITLE
Decreased dendritic spine number 

### ISSUE THREAD

**Issue by sarahstofel** [obophenotype/human-phenotype-ontology#7056]
**Preferred term label:**
Decreased dendritic spine number 

**Synonyms**

**Definition (free text, please give PubMed ID)**
Reduction in the amount of spines on a given dendrite.
PMID:8046469

**Parent term (use hpo.jax.org/app)**

abnormal neuron branching

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

OMIM: 104300

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0003-2307-1226

**Comment by pnrobinson** [comment]
added term as HP_0034005
